com.unity.multiplayer.mlapi/Runtime/AssemblyInfo.cs

```diff
@@ -5,4 +5,5 @@
 [assembly: InternalsVisibleTo("Unity.Multiplayer.MLAPI.RuntimeTests")]
 [assembly: InternalsVisibleTo("Unity.Multiplayer.MLAPI.Editor.CodeGen")]
 [assembly: InternalsVisibleTo("Unity.Multiplayer.MLAPI.Editor")]
+[assembly: InternalsVisibleTo("Unity.Multiplayer.MLAPI.TestProject.UnitTests.Runtime")]
```

Comment:
to keep things super-similar to others like `Unity.Multiplayer.MLAPI.RuntimeTests`
I'd suggest naming testproject asmdef as `Unity.Multiplayer.MLAPI.TestProject.RuntimeTests`